src/content/docs/blog/2025/06/30/mid-year-update/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Mid Year Update"
+date: 2025-06-30T21:37:00Z
+authors: [nomadiccore,ermo]
+tags: [news]
+---
+import Kofi from "@/components/ui/Kofi.astro";
+
+As we hit the middle of the year, it's time for another update for those of you following along with AerynOS's development.
+
+Over the last few months, things may have seemed unusually quiet, however rest assured that there has been A LOT going on in the background. As such, we are preparing a short series of blog posts to go over the relevant topics in the coming weeks.
+
+For this blog post, we are going to cover our infrastructure port, along with the process of rebuilding our entire package repository.
+
+**The TL;DR is that:**
+
+- All core AerynOS tooling is now written in Rust
+- Every recipe in the repository has been rebuilt (twice!) with many packages then having been updated to newer versions after the rebuilds were completed
+- A CDN has been implemented for faster package installation and ISO downloads
+
+## New Infrastructure tooling
+
+When delivering a Linux distribution, its infrastructure and associated processes effectively act as the "spine" of the project. But spine surgery can be a delicate affair, particularly when it comes to rehabilitation after successful surgery.
+
+For us, this cycle has been particularly demanding, as we have completed an MVP port of our infrastructure tooling code to Rust, meaning that all core AerynOS tooling has now fully transitioned away from DLang.
+
+We have covered the reasons for transitioning away from DLang [previously](https://aerynos.com/blog/2023/09/06/oxidised-moss/), and it's fair to say that we are already feeling the benefits of easy and native reuse of code in our tooling repositories and welcoming more Rust contributors into our community.
```

Comment:
I'd say leave out the repeated "from DLang" or even the whole "away from DLang" since it's already written the same way in the previous sentence.